src/sm64ap.cpp

```diff
@@ -275,6 +283,13 @@ void SM64AP_SetMoveRandoVec(int vec) {
         sm64_have_abilities[i] = !std::bitset<SM64AP_NUM_ABILITIES>(vec).test(i) || sm64_have_abilities[i];
     }
 }
+void SM64AP_SetPaintingRando(int boolAsInt) {
+    if(boolAsInt == 0) { // 0 = not enabled, so unlock all paintings
```

Comment:
`boolAsInt` is a technical detail that isnt really relevant. Just call it `enable` or something like that.
Also, you can just do `if (!boolAsInt)`